### ISSUE TITLE
no -show option available for h5fc 

### ISSUE THREAD

**Issue by cd10kfsu** [HDFGroup/hdf5#4574]
**Describe the bug**
I used to run `h5fc -show test.f90` to display the full compiling & linking commands with hdf5 libs in Fortran. But for the home-brew installed HDF 1.14.3, option `-show` is no longer available. 
```
cda14@chengs14 test % h5fc -show a.f90
/opt/homebrew/h5fc
dir is /opt/homebrew
gfortran: error: unrecognized command-line option '-show'
```

Is this a bug, or the script `h5fc` no longer include `-show` option? If so, is there an equivalent option as `-show`? 

**Expected behavior**
Using an older version (1.8.22) of hdf5, I have
```
cda14@chengs14 bin % ./h5fc -show a.f90
gfortran-13 -I/Users/cda14/Documents/model/mpas_gnu13/baselibs/hdf5_1.8.22/include a.f90 -L/Users/cda14/Documents/model/mpas_gnu13/baselibs/hdf5_1.8.22/lib /Users/cda14/Documents/model/mpas_gnu13/baselibs/hdf5_1.8.22/lib/libhdf5hl_fortran.a /Users/cda14/Documents/model/mpas_gnu13/baselibs/hdf5_1.8.22/lib/libhdf5_hl.a /Users/cda14/Documents/model/mpas_gnu13/baselibs/hdf5_1.8.22/lib/libhdf5_fortran.a /Users/cda14/Documents/model/mpas_gnu13/baselibs/hdf5_1.8.22/lib/libhdf5.a -L/Users/cda14/Documents/model/mpas_gnu13/baselibs/zlib_1.2.7/lib -lz -ldl -lm
```
**Platform (please complete the following information)**
 - HDF5 version: 1.14.3 (installed with homebrew)
 - OS and version: macOS Monterey, Version 12.7.1
 - Compiler and version: gcc-13
 - Build system (e.g. CMake, Autotools) and version
 - Any configure options you specified
 - MPI library and version (parallel HDF5): non-MPI version 

**Additional context**

**Comment by brtnfld** [comment]
It probably works with 1.8 in Homebrew because it uses autotools to build HDF5. However, Homebrew has switched to CMake builds, which lack Autotools' options for the h5 wrappers. I'll create an issue to address that.

**Comment by fortnern** [comment]
Closing this issue, as it's included in #4582
